autoload/utils/cmake.vim

```diff
@@ -188,3 +188,29 @@ function! utils#cmake#getBuildDir() abort
     let l:build_dir = fnamemodify(l:build_dir, ':p:h')
     return l:build_dir
 endfunction
+
+function! utils#cmake#getBinaryPath() abort
+    let l:cmake_info = utils#cmake#common#getInfo()
+    if has_key(l:cmake_info, 'targets') && has_key(l:cmake_info['targets'], g:cmake_build_target)
+        let l:target = l:cmake_info['targets'][g:cmake_build_target]
+        if l:target['type'] !=# 'EXECUTABLE'
+            let v:errmsg = 'Target ' . g:cmake_build_target . 'is not an executable'
+            call utils#common#Warning(v:errmsg)
+            return ''
+        endif
+        if has('win32')
+            return utils#fs#fnameescape(utils#cmake#getBuildDir() . '\' . l:target['pathes'][0])
+        else
+            return utils#fs#fnameescape(utils#cmake#getBuildDir() . '/' . l:target['pathes'][0])
```

Comment:
Path can be absolute if I specify bin folder and it is outside the build directory. But I can fix it later.